src/Service.Tests/SqlTests/RestApiTests/Find/FindApiTestBase.cs

```diff
@@ -447,6 +447,26 @@ public async Task FindTestWithFirstSingleKeyPagination()
             );
         }
 
+        /// <summary>
+        /// Validates that a proper nextLink is created for FindMany requests which do not
+        /// restrict results with query parameters. Engine default paging mechanisms are used
+        /// when > 100 records will be present in result set.
+        /// expectedAfterQueryString starts with ?$, and not &$, because it is the only query parameter.
```

Comment:
If it were not the only query parameter does it start with `&$` and is there a test already for that case?

Reply:
added test coverage for this to Add coverage test for scenario when orderby is one of many query params and is also not the first. Which checks for alternative prefix to after query param: &$ and not ?$